packages/core/src/forms/forms.stories.ts

```diff
@@ -1851,7 +1851,7 @@ export const responsiveCheckoutForm = () => {
     const iframe = document.querySelector<HTMLIFrameElement>('#complex-form-demo');
     iframe.onload = () => {
       const resizeObserver = new ResizeObserver(
-        () => (iframe.style.height = `${iframe.contentWindow.document.body.scrollHeight}px`)
+        () => (iframe.style.height = `${iframe.contentWindow?.document.body.scrollHeight}px`)
```

Comment:
Depending on the browser sometimes storybook loads this in the background and can cause errors if not checked